internal/pkg/v2/infrastructure/redis/dbcommands.go

```diff
@@ -32,4 +34,10 @@ const (
 	InfiniteMin     = "-inf"
 	InfiniteMax     = "+inf"
 	GreaterThanZero = "(0"
+	DBKeySeparator  = ":"
 )
+
+// CreateKey creates Redis key by connecting the target key with DBKeySeparator
+func CreateKey(targets ...string) string {
```

Comment:
I'd suggest to rename `dbcommnds.go` to` dbconsts.go`, and move `func CreateKey` to an independent `dbhelper.go` file.  @cloudxxx8 and @lenny-intel, what's your thought on this?

Reply:
yes, it would be better

Reply:
Renamed dbcommands.go to dbconsts.go and create dbhelper.go